### ISSUE TITLE
IC potential clash with jquery ?

### ISSUE THREAD

**Issue by Arico1985** [intercoolerjs/intercooler-js#309]
Hello hello - Do 'ic-post-to' remove or clashes with jquery ? I presume not & I must be doing something wrong somewhere, but just checking.
In more detail, I have a template from which I can create new object instances using IC, Template A:
```
<div id="page-content">
[stuff]
  <div>
     <a
     ic-get-from="{% url 'home' %}?add_new=True"
     ic-target="#page-new" >
      <i class='fa fa-plus'></i>
     </a>
  </div>

  <div id="page-new"></div>

</div>
```

and an other one that handle the actual creation of the new object instance, Template B:
```
<form ic-post-to="{% url 'home' %}?add_new=True"
method="post"
ic-target="#page-content"
ic-select-from-response="#page-content">

  <div>{{new_object.name}}</div>
  <button type="submit" >Submit</button>
</form>
```

So far so good. Now I want to add my own js and added on Template A an event listener on the div '#js-test':
```
<div id="js-test"></div>
```

and the js:
```
$(document).ready(function() {
  console.log('js called !');
  $('#js-test').click(function(){
    console.log('Math.random():', Math.random())
  });
});
```

Now, whenever I create new objects and refresh the page, the js works. If I put the listener outside the div `page-content`, it also works. But as soon as I create a new object with IC, if I keep the listener inside the element targeted by IC , the div `page-content`, then the console doesn't even log 'js called'.

in other words this works:
```
<div id="page-content">
[stuff]
  <div>
     <a
     ic-get-from="{% url 'home' %}?add_new=True"
     ic-target="#page-new" >
      <i class='fa fa-plus'></i>
     </a>
  </div>

  <div id="page-new"></div>

</div>

<div id="js-test"></div>
```

but this doesn't:
```
<div id="page-content">
[stuff]
  <div>
     <a
     ic-get-from="{% url 'home' %}?add_new=True"
     ic-target="#page-new" >
      <i class='fa fa-plus'></i>
     </a>
  </div>

  <div id="page-new"></div>

  <div id="js-test"></div>

</div>
```

Any obvious mistakes to you ?

**Comment by Arico1985** [comment]
Rhaaa super sweet, thank you !!

**Comment by Arico1985** [comment]
:) :) :) :) :) :) awesome, thank you !!!!
